storehaus-redis/src/main/scala/com/twitter/storehaus/redis/RedisStore.scala

```diff
@@ -34,24 +40,33 @@ object RedisStore {
 
   def apply(client: Client, ttl: Option[Duration] = Default.TTL): RedisStore =
     new RedisStore(client, ttl)
+
+  def toChannelBufferStore(store: Store[Buf, Buf]) : Store[ChannelBuffer, ChannelBuffer] = {
+    store.convert(ChannelBuffer2BufInjection.apply)(ChannelBuffer2BufInjection)
+  }
+}
+
+object ChannelBuffer2BufInjection extends Injection[ChannelBuffer, Buf] {
```

Comment:
this should be a bijection, not an Injection (there is an implicit from Bijection to Injection, by the way so you can pass this as the injection above).